internal/util/state/state.go

```diff
@@ -21,14 +21,14 @@ import (
 	"github.com/AlekSi/pointer"
 	"github.com/google/uuid"
 
-	"github.com/FerretDB/FerretDB/build/version"
 	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
 // State represents FerretDB process state.
 type State struct {
-	UUID      string `json:"uuid"`
-	Telemetry *bool  `json:"telemetry,omitempty"` // nil for undecided
+	UUID              string `json:"uuid"`
+	Telemetry         *bool  `json:"telemetry,omitempty"` // nil for undecided
+	IsUpdateAvailable bool   `json:"update_available"`
```

Comment:
Got this from beacon https://github.com/FerretDB/beacon/blob/main/internal/handler.go#L58, maybe not correct.